### ISSUE TITLE
Deno demo deploy fail

### ISSUE THREAD

**Issue by hu-qi** [Tencent/cloudbase-framework#166]
**Bug 描述**
**Describe the bug**
Deno 的案例部署失败。

请简单清晰地描述您遇到的bug。
我通过`tcb new deno  deno`通过模板新建一个 deno 项目，
然后执行`tcb framework deploy` 进行一键部署就失败！

**重现步骤**
**To Reproduce**

重现 bug 的步骤：
1. 执行 'tcb new deno  deno'
2. 执行'cd deno && tcb new deno  deno'
3. 看到报错信息

**预期的行为**
**Expected behavior**
部署成功

**截图**
**Screenshots**
![image](https://user-images.githubusercontent.com/17986122/105666742-e6237480-5f14-11eb-8f64-4654658aa4ef.png)

**错误日志**
**Debug LOG**
```log
gpdi@Mac-mini deno % tcb framework deploy
CloudBase CLI 1.2.7
CloudBase Framework 1.6.11
   ______ __                   __ ____                             
  / ____// /____   __  __ ____/ // __ ) ____ _ _____ ___           
 / /    / // __ \ / / / // __  // __  |/ __ `// ___// _ \          
/ /___ / // /_/ // /_/ // /_/ // /_/ // /_/ /(__  )/  __/          
\_________\____/ \__,_/ \__,_//_____/ \__,_//____/ \___/       __  
   / ____/_____ ____ _ ____ ___   ___  _      __ ____   _____ / /__
  / /_   / ___// __ `// __ `__ \ / _ \| | /| / // __ \ / ___// //_/
 / __/  / /   / /_/ // / / / / //  __/| |/ |/ // /_/ // /   / ,<   
/_/    /_/    \__,_//_/ /_/ /_/ \___/ |__/|__/ \____//_/   /_/|_|  
                                                                   

 CloudBase Framework  info     Version v1.6.11
 CloudBase Framework  info     Github: https://github.com/Tencent/cloudbase-framework

 CloudBase Framework  info     EnvId deno-bot-5gvcin9606db0bc5
 CloudBase Framework  info     Validate config file success.
 CloudBase Framework  info     AppName deno-app
 CloudBase Framework  info     📦 install plugins
 CloudBase Framework  info     callHooks 'preDeploy' 
 CloudBase Framework  info     🔧 init: deno...
 CloudBase Framework  info     🔨 build: deno...
 CloudBase Framework  info     🧬 compile: deno...
 CloudBase Framework  info     callHooks 'postCompile' 
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░] 100% 25.0 s
 CloudBase Framework  error    [DEPLOY_ERROR]  
部署失败，错误信息：1344533_DescribeCloudBaseRunServer_[800003][800003]Docker build ...
Sending build context to Docker daemon  820.2kB
Step 1/14 : FROM debian:latest
latest: Pulling from library/debian
b9a857cbf04d: Pulling fs layer
b9a857cbf04d: Verifying Checksum
b9a857cbf04d: Download complete
b9a857cbf04d: Pull complete
Digest: sha256:b16f66714660c4b3ea14d273ad8c35079b81b35d65d1e206072d226c7ff78299
Status: Downloaded newer image for debian:latest
 ---> e7d08cddf791
Step 2/14 : WORKDIR /install
 ---> Running in 97451822a483
Removing intermediate container 974尝试 重新构建;查看 构建详情;检查Dockerfile 是否位于正确位置: ./Dockerfile;提交工单，并在 问题描述 中附上以下信息：环境id：deno-bot-5gvcin9606db0bc5；服务名称：deno-app；版本：deno-app-003；状态：构建失败;， 请求RequestId：be13540c-0495-451e-9661-9402aaf16231
 CloudBase Framework  info     部署日志: '/Users/gpdi/cloudbase-framework/logs/2021-01-25_13-51-54.log' 
gpdi@Mac-mini deno % tcb framework deploy --verbose
CloudBase CLI 1.2.7
CloudBase Framework 1.6.11
   ______ __                   __ ____                             
  / ____// /____   __  __ ____/ // __ ) ____ _ _____ ___           
 / /    / // __ \ / / / // __  // __  |/ __ `// ___// _ \          
/ /___ / // /_/ // /_/ // /_/ // /_/ // /_/ /(__  )/  __/          
\_________\____/ \__,_/ \__,_//_____/ \__,_//____/ \___/       __  
   / ____/_____ ____ _ ____ ___   ___  _      __ ____   _____ / /__
  / /_   / ___// __ `// __ `__ \ / _ \| | /| / // __ \ / ___// //_/
 / __/  / /   / /_/ // / / / / //  __/| |/ |/ // /_/ // /   / ,<   
/_/    /_/    \__,_//_/ /_/ /_/ \___/ |__/|__/ \____//_/   /_/|_|  
                                                                   

 CloudBase Framework  info     Version v1.6.11
 CloudBase Framework  info     Github: https://github.com/Tencent/cloudbase-framework

 CloudBase Framework  info     EnvId deno-bot-5gvcin9606db0bc5
 CloudBase Framework  debug    本地构建，本地存在配置文件 {
  functionRoot: './functions',
  functions: [],
  version: '2.0',
  envId: 'deno-bot-5gvcin9606db0bc5',
  '$schema': 'https://framework-1258016615.tcloudbaseapp.com/schema/latest.json',
  framework: {
    name: 'deno-app',
    plugins: {
      deno: {
        use: '@cloudbase/framework-plugin-deno',
        inputs: { dockerImage: 'debian:latest' }
      }
    }
  },
  region: 'ap-shanghai'
} 
 CloudBase Framework  debug    远程存在同名项目 'deno-app' 
 CloudBase Framework  debug    项目配置信息 'rcConfig' {
  functionRoot: './functions',
  functions: [],
  version: '2.0',
  envId: 'deno-bot-5gvcin9606db0bc5',
  '$schema': 'https://framework-1258016615.tcloudbaseapp.com/schema/latest.json',
  framework: {
    name: 'deno-app',
    plugins: {
      deno: {
        use: '@cloudbase/framework-plugin-deno',
        inputs: { dockerImage: 'debian:latest' }
      }
    }
  },
  region: 'ap-shanghai'
} 'extraData' {
  repo: {
    Type: '',
    Url: 'https://gitee.com/hu-qi/trending-in-one-tcb',
    Name: 'deno-app',
    WorkDir: '',
    RawCode: '',
    Headers: {}
  },
  tags: [],
  environment: {},
  network: {},
  addons: {}
} 'projectName' 'deno-app' 
 CloudBase Framework  debug    RC 配置文件变更 false {
  functionRoot: './functions',
  functions: [],
  version: '2.0',
  envId: 'deno-bot-5gvcin9606db0bc5',
  '$schema': 'https://framework-1258016615.tcloudbaseapp.com/schema/latest.json',
  framework: {
    name: 'deno-app',
    plugins: {
      deno: {
        use: '@cloudbase/framework-plugin-deno',
        inputs: { dockerImage: 'debian:latest' }
      }
    }
  },
  region: 'ap-shanghai'
} 
 CloudBase Framework  debug    Framework 配置文件变更 false {
  name: 'deno-app',
  plugins: {
    deno: {
      use: '@cloudbase/framework-plugin-deno',
      inputs: { dockerImage: 'debian:latest' }
    }
  }
} 
 CloudBase Framework  info     Validate config file success.
 CloudBase Framework  info     AppName deno-app
 CloudBase Framework  debug    appConfig {
  name: 'deno-app',
  plugins: {
    deno: {
      use: '@cloudbase/framework-plugin-deno',
      inputs: { dockerImage: 'debian:latest' }
    }
  },
  repo: {
    Type: '',
    Url: 'https://gitee.com/hu-qi/trending-in-one-tcb',
    Name: 'deno-app',
    WorkDir: '',
    RawCode: '',
    Headers: {}
  },
  tags: [],
  environment: {},
  network: {},
  addons: {}
} 
 CloudBase Framework  info     📦 install plugins
 CloudBase Framework  debug    initHooks {} 
 CloudBase Framework  debug    resolve plugin @cloudbase/framework-plugin-deno@1.6.11: use 0.00 s, mismatch：false, resolved version：1.6.11
 CloudBase Framework  debug    deploy '' '' 
 CloudBase Framework  info     callHooks 'preDeploy' 
 CloudBase Framework  info     🔧 init: deno...
 CloudBase Framework  debug    DenoPlugin: init {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.'
} 
 CloudBase Framework  info     🔨 build: deno...
 CloudBase Framework  debug    DenoPlugin: build {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.'
} 
 CloudBase Framework  debug    ContainerPlugin: init {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.',
  localAbsolutePath: '/Users/gpdi/cloudbase-framework/builds/cloudbase-deno-build-1611554114477/deno-app'
} 
 CloudBase Framework  debug    Container Plugin Inputs 'cloud' { cpu: 0.25, mem: 0.5, maxNum: 10, minNum: 0 } 'mode' {} 
 CloudBase Framework  debug    ContainerPlugin: build {
  uploadType: 'package',
  description: '基于云开发 CloudBase Framework 部署的云托管',
  isPublic: true,
  flowRatio: 100,
  cpu: 0.25,
  mem: 0.5,
  minNum: 0,
  maxNum: 10,
  policyType: 'cpu',
  policyThreshold: 60,
  containerPort: 80,
  dockerfilePath: './Dockerfile',
  buildDir: './',
  version: '1.0.0',
  localPath: './',
  envVariables: {},
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.',
  localAbsolutePath: '/Users/gpdi/cloudbase-framework/builds/cloudbase-deno-build-1611554114477/deno-app'
} 
 CloudBase Framework  debug    describeCloudBaseBuildService {
  PackageName: 'deno-app.zip',
  PackageVersion: '1611554115',
  RequestId: '1bc061d6-a443-4711-a756-9863c7a41adf',
  UploadHeaders: [],
  UploadUrl: 'https://cloudaccess-run-1258016615.cos.ap-shanghai.myqcloud.com/home%2F922d0822225540ed9db967c5ef2bb198%2Fdeno-bot-5gvcin9606db0bc5-deno-app-_1611554115_8r7Sf4nl.zip?sign=q-sign-algorithm%3Dsha1%26q-ak%3DAKIDYqreAvj0McQ8bplDBtRUJcfwwlL299NL%26q-sign-time%3D1611554115%3B1612158915%26q-key-time%3D1611554115%3B1612158915%26q-header-list%3Dhost%26q-url-param-list%3D%26q-signature%3D9b5bf32b01fca8caca206a631f5f0495a7770d23'
} 'deno-app' '/Users/gpdi/cloudbase-framework/builds/cloudbase-container-build-1611554114513/deno-app.zip' 
 CloudBase Framework  info     🧬 compile: deno...
 CloudBase Framework  debug    DenoPlugin: compile {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.'
} 
 CloudBase Framework  debug    ContainerPlugin: compile {
  uploadType: 'package',
  description: '基于云开发 CloudBase Framework 部署的云托管',
  isPublic: true,
  flowRatio: 100,
  cpu: 0.25,
  mem: 0.5,
  minNum: 0,
  maxNum: 10,
  policyType: 'cpu',
  policyThreshold: 60,
  containerPort: 80,
  dockerfilePath: './Dockerfile',
  buildDir: './',
  version: '1.0.0',
  localPath: './',
  envVariables: {},
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.',
  localAbsolutePath: '/Users/gpdi/cloudbase-framework/builds/cloudbase-deno-build-1611554114477/deno-app'
} 
 CloudBase Framework  info     callHooks 'postCompile' 
 CloudBase Framework  debug    hooks.genSAM {} 
 CloudBase Framework  debug    sam:install {
  TCBSAMFormatVersion: '2020-06-15',
  License: 'MIT',
  Description: '基于 CloudBase Framework 构建',
  Category: 'CloudApp',
  Tags: [],
  SourceUrl: 'https://github.com/Tencent/cloudbase-framework',
  ReleaseNotesUrl: 'https://github.com/Tencent/cloudbase-framework/releases',
  IconUrl: 'https://main.qcloudimg.com/raw/f87b49914efe502722f9aa128f5d8865/hello_world.svg',
  IntroUrl: 'https://cloud.tencent.com/document/product/876/44547',
  ChangeLog: [ { Title: '1.0.0', Content: [ '初次发布' ] } ],
  Author: {
    AuthorName: 'Tencent CloudBase',
    Email: 'cloudbase@tencent.com',
    Url: 'https://github.com/Tencent/cloudbase-framework'
  },
  Contributors: [
    {
      AuthorName: 'Tencent CloudBase',
      Email: 'cloudbase@tencent.com',
      Url: 'https://github.com/Tencent/cloudbase-framework'
    }
  ],
  Name: 'deno-app',
  Version: '1.0.0',
  DisplayName: 'deno-app',
  Globals: { Environment: { Variables: {} } },
  SourceDir: '.',
  Resources: {
    DenoApp: {
      Type: 'CloudBase::CloudBaseRun',
      Properties: {
        ServerName: 'deno-app',
        Description: '基于云开发 CloudBase Framework 部署的云托管',
        isPublic: true,
        UploadType: 'package',
        FlowRatio: 100,
        Cpu: 0.25,
        Mem: 0.5,
        MinNum: 0,
        MaxNum: 10,
        PolicyType: 'cpu',
        PolicyThreshold: 60,
        ContainerPort: 80,
        DockerfilePath: './Dockerfile',
        BuildDir: './',
        HttpPath: '/deno-app',
        EnvParams: '{}',
        PackageName: 'deno-app.zip',
        PackageVersion: '1611554115'
      }
    }
  },
  EntryPoint: [
    {
      Label: '服务入口',
      EntryType: 'HttpService',
      HttpEntryPath: '/deno-app'
    }
  ]
} 
 CloudBase Framework  debug    ext taskInfo {
  ExtensionId: 'deno-app-HkrDtpIb',
  Status: 'installing',
  Detail: '',
  Percent: 75
} 
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░          ] 75% 0.0 s CloudBase Framework  debug    ext taskInfo {
  ExtensionId: 'deno-app-HkrDtpIb',
  Status: 'installing',
  Detail: '',
  Percent: 75
} 
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░          ] 75% 5.0 s CloudBase Framework  debug    ext taskInfo {
  ExtensionId: 'deno-app-HkrDtpIb',
  Status: 'installing',
  Detail: '',
  Percent: 75
} 
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░          ] 75% 10.0 s CloudBase Framework  debug    ext taskInfo {
  ExtensionId: 'deno-app-HkrDtpIb',
  Status: 'installing',
  Detail: '',
  Percent: 75
} 
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░          ] 75% 15.0 s CloudBase Framework  debug    ext taskInfo {
  ExtensionId: 'deno-app-HkrDtpIb',
  Status: 'installError',
  Detail: '1344537_DescribeCloudBaseRunServer_[800003][800003]Docker build ...\n' +
    'Sending build context to Docker daemon  820.2kB\r\r\n' +
    'Step 1/14 : FROM debian:latest\n' +
    'latest: Pulling from library/debian\n' +
    'b9a857cbf04d: Pulling fs layer\n' +
    'b9a857cbf04d: Verifying Checksum\n' +
    'b9a857cbf04d: Download complete\n' +
    'b9a857cbf04d: Pull complete\n' +
    'Digest: sha256:b16f66714660c4b3ea14d273ad8c35079b81b35d65d1e206072d226c7ff78299\n' +
    'Status: Downloaded newer image for debian:latest\n' +
    ' ---> e7d08cddf791\n' +
    'Step 2/14 : WORKDIR /install\n' +
    ' ---> Running in 31d8f574f368\n' +
    'Removing intermediate container 31d尝试 重新构建;查看 构建详情;检查Dockerfile 是否位于正确位置: ./Dockerfile;提交工单，并在 问题描述 中附上以下信息：环境id：deno-bot-5gvcin9606db0bc5；服务名称：deno-app；版本：deno-app-004；状态：构建失败;',
  Percent: 100
} 
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░] 100% 20.0 s
 CloudBase Framework  error    [DEPLOY_ERROR]  
部署失败，错误信息：1344537_DescribeCloudBaseRunServer_[800003][800003]Docker build ...
Sending build context to Docker daemon  820.2kB
Step 1/14 : FROM debian:latest
latest: Pulling from library/debian
b9a857cbf04d: Pulling fs layer
b9a857cbf04d: Verifying Checksum
b9a857cbf04d: Download complete
b9a857cbf04d: Pull complete
Digest: sha256:b16f66714660c4b3ea14d273ad8c35079b81b35d65d1e206072d226c7ff78299
Status: Downloaded newer image for debian:latest
 ---> e7d08cddf791
Step 2/14 : WORKDIR /install
 ---> Running in 31d8f574f368
Removing intermediate container 31d尝试 重新构建;查看 构建详情;检查Dockerfile 是否位于正确位置: ./Dockerfile;提交工单，并在 问题描述 中附上以下信息：环境id：deno-bot-5gvcin9606db0bc5；服务名称：deno-app；版本：deno-app-004；状态：构建失败;， 请求RequestId：67130ebc-6764-4e57-86c6-e09050bd3900
```

**其他上下文信息**
**Additional context**

之前咨询过技术工单，反馈是网络的问题，感觉不太像！

**Comment by binggg** [comment]
curl -fsSL https://x.deno.js.cn/install.sh这里在云端拉取有问题

后端在查了

**Comment by hu-qi** [comment]
> curl -fsSL [https://x.deno.js.cn/install.sh这里在云端拉取有问题](https://x.deno.js.cn/install.sh%E8%BF%99%E9%87%8C%E5%9C%A8%E4%BA%91%E7%AB%AF%E6%8B%89%E5%8F%96%E6%9C%89%E9%97%AE%E9%A2%98)
> 
> 后端在查了

是的 之前提示 403

**Comment by binggg** [comment]
![image](https://user-images.githubusercontent.com/7686861/105854920-70a5ca00-6022-11eb-84af-0d283f261d1a.png)
![Uploading image.png…]()

**Comment by hu-qi** [comment]
这是因为 deno 镜像超过 20 M 的限制了吗？之前听说 jsdelivr 的限制是 20 M

**Comment by hu-qi** [comment]
还是有点小问题：
```
huqi@huqi MINGW64 /f/dosomethings/deno/deno_demo
$ tcb
CloudBase CLI 1.3.0
CloudBase Framework 1.6.13
√ 是否使用云开发部署当前项目 <F:\dosomethings\deno\deno_demo> ？ (Y/n) · true
   ______ __                   __ ____                             
  / ____// /____   __  __ ____/ // __ ) ____ _ _____ ___           
 / /    / // __ \ / / / // __  // __  |/ __ `// ___// _ \          
/ /___ / // /_/ // /_/ // /_/ // /_/ // /_/ /(__  )/  __/          
\_________\____/ \__,_/ \__,_//_____/ \__,_//____/ \___/       __  
   / ____/_____ ____ _ ____ ___   ___  _      __ ____   _____ / /__
  / /_   / ___// __ `// __ `__ \ / _ \| | /| / // __ \ / ___// //_/
 / __/  / /   / /_/ // / / / / //  __/| |/ |/ // /_/ // /   / ,<   
/_/    /_/    \__,_//_/ /_/ /_/ \___/ |__/|__/ \____//_/   /_/|_|  
                                                                   

 CloudBase Framework  info     Version v1.6.13
 CloudBase Framework  info     Github: https://github.com/Tencent/cloudbase-framework

 CloudBase Framework  info     EnvId deno-demo-3ghgaf7pa8d97b90
 CloudBase Framework  info     Validate config file success.
 CloudBase Framework  info     AppName deno-app
 CloudBase Framework  info     ◆ install plugins
 CloudBase Framework  info     callHooks 'preDeploy'
+ @cloudbase/framework-plugin-deno@1.6.13
added 417 packages from 427 contributors in 106.024s
 CloudBase Framework  info     ◆ init: deno...
 CloudBase Framework  info     ◆ build: deno...
 CloudBase Framework  info     ◆ compile: deno...
 CloudBase Framework  info     callHooks 'postCompile'
正在部署[░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░] 100% 25.1 s
 CloudBase Framework  error    [DEPLOY_ERROR]
部署失败，错误信息：1386707_DescribeCloudBaseRunServer_[800003][800003]Docker build ...
Sending build context to Docker daemon  820.2kB
Step 1/14 : FROM debian:latest
latest: Pulling from library/debian
b9a857cbf04d: Pulling fs layer
b9a857cbf04d: Verifying Checksum
b9a857cbf04d: Download complete
b9a857cbf04d: Pull complete
Digest: sha256:b16f66714660c4b3ea14d273ad8c35079b81b35d65d1e206072d226c7ff78299
Status: Downloaded newer image for debian:latest
 ---> e7d08cddf791
Step 2/14 : WORKDIR /install
 ---> Running in d420eeb33f8d
Removing intermediate container d42尝试 重新构建;查看 构建详情;检查Dockerfile 是否位于正确位置: ./Dockerfile;提交工单，并在 问题描述 中附上以下信息：环境id：deno-demo-3ghgaf7pa8d97b90；服务名称：deno-app；版本：deno-app-002；状 
态：构建失败;， 请求RequestId：9266a7de-c3bf-43eb-ab26-4f589324d074
 CloudBase Framework  info     部署日志: 'C:\\Users\\huqi\\cloudbase-framework\\logs\\2021-01-26_23-56-41.log'
```
需要新建一个 Dockerfile 吗？

**Comment by binggg** [comment]
应该是同一个问题

**Comment by hu-qi** [comment]
> 应该是同一个问题

目前本地安装 Deno 是正常的
```
gpdi@Mac-mini gpdi-admin % curl -fsSL https://x.deno.js.cn/install.sh | sh
######################################################################## 100.0%
Archive:  /Users/gpdi/.deno/bin/deno.zip
  inflating: /Users/gpdi/.deno/bin/deno  
Deno 已经成功安装
可执行文件位置为 /Users/gpdi/.deno/bin/deno
运行 'deno --help' 查看 Deno 帮助信息
gpdi@Mac-mini gpdi-admin % 
gpdi@Mac-mini gpdi-admin % deno -V
deno 1.7.0
gpdi@Mac-mini gpdi-admin % 
```

**Comment by binggg** [comment]
![image](https://user-images.githubusercontent.com/7686861/105937734-21e34900-6091-11eb-983a-17c257918aae.png)

**Comment by binggg** [comment]
在 deno 社区没有修复之前，可以在配置文件中指定 deno runtime 为 1.6.3

```json
{
  "version": "2.0",
  "envId": "{{envId}}",
  "$schema": "https://framework-1258016615.tcloudbaseapp.com/schema/latest.json",
  "framework": {
    "name": "deno-app",
    "plugins": {
      "deno": {
        "use": "@cloudbase/framework-plugin-deno",
        "inputs": {
          "dockerImage": "debian:latest",
          "runtime": "1.6.3"
        }
      }
    }
  }
}

```
![image](https://user-images.githubusercontent.com/7686861/105938874-0d07b500-6093-11eb-9d9c-8bb1ee835789.png)

**Comment by hu-qi** [comment]
![image](https://user-images.githubusercontent.com/17986122/105940777-01b68880-6097-11eb-93a3-d646b3d65332.png)

可以了，感谢！

**Comment by hu-qi** [comment]
好像又有了新的问题：
```
 CloudBase Framework  info     Version v1.6.15
 CloudBase Framework  info     Github: https://github.com/Tencent/cloudbase-framework

 CloudBase Framework  info     EnvId deno-demo-3ghgaf7pa8d97b90
 CloudBase Framework  info     Validate config file success.
 CloudBase Framework  info     AppName deno-app
 CloudBase Framework  info     📦 install plugins
 CloudBase Framework  info     callHooks 'preDeploy' 
 CloudBase Framework  info     🔧 init: deno...
 CloudBase Framework  info     🔨 build: deno...
 CloudBase Framework  error     Cannot read property 'reduce' of null
 CloudBase Framework  info     部署日志: '/Users/gpdi/cloudbase-framework/logs/2021-01-29_16-31-45.log' 
```

**Comment by binggg** [comment]
@hu-qi 部署日志看看呢

**Comment by hu-qi** [comment]
> @hu-qi 部署日志看看呢

Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 4.7 CloudBase Framework::info Version [32mv1.6.15[39m
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 4.7 CloudBase Framework::info Github: [1m[4m[36mhttps://github.com/Tencent/cloudbase-framework[39m[24m[22m

Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 4.7 CloudBase Framework::info EnvId [32mdeno-demo-3ghgaf7pa8d97b90[39m
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 4.7 CloudBase Framework::debug 本地构建，本地存在配置文件 {
  functionRoot: './functions',
  functions: [],
  envId: 'deno-demo-3ghgaf7pa8d97b90',
  version: '2.0',
  '$schema': 'https://framework-1258016615.tcloudbaseapp.com/schema/latest.json',
  region: 'ap-shanghai',
  framework: {
    name: 'versailles',
    plugins: {
      deno: {
        use: '@cloudbase/framework-plugin-deno',
        inputs: { dockerImage: 'debian:latest' }
      }
    }
  }
} 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::debug 远程不存在同名项目，新建项目
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::debug 项目配置信息 'rcConfig' {
  functionRoot: './functions',
  functions: [],
  envId: 'deno-demo-3ghgaf7pa8d97b90',
  version: '2.0',
  '$schema': 'https://framework-1258016615.tcloudbaseapp.com/schema/latest.json',
  region: 'ap-shanghai',
  framework: {
    name: 'versailles',
    plugins: {
      deno: {
        use: '@cloudbase/framework-plugin-deno',
        inputs: { dockerImage: 'debian:latest' }
      }
    }
  }
} 'extraData' {} 'projectName' 'versailles' 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::debug RC 配置文件变更 false {
  functionRoot: './functions',
  functions: [],
  envId: 'deno-demo-3ghgaf7pa8d97b90',
  version: '2.0',
  '$schema': 'https://framework-1258016615.tcloudbaseapp.com/schema/latest.json',
  region: 'ap-shanghai',
  framework: {
    name: 'versailles',
    plugins: {
      deno: {
        use: '@cloudbase/framework-plugin-deno',
        inputs: { dockerImage: 'debian:latest' }
      }
    }
  }
} 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::debug Framework 配置文件变更 false {
  name: 'versailles',
  plugins: {
    deno: {
      use: '@cloudbase/framework-plugin-deno',
      inputs: { dockerImage: 'debian:latest' }
    }
  }
} 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::info Validate config file success.
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::info AppName [32mversailles[39m
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.0 CloudBase Framework::debug appConfig {
  name: 'versailles',
  plugins: {
    deno: {
      use: '@cloudbase/framework-plugin-deno',
      inputs: { dockerImage: 'debian:latest' }
    }
  }
} 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.2 CloudBase Framework::info 📦 install plugins
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.2 CloudBase Framework::debug initHooks {} 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.2 CloudBase Framework::debug resolve plugin @cloudbase/framework-plugin-deno@1.6.15: use 0.00 s, mismatch：false, resolved version：1.6.15
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.2 CloudBase Framework::debug deploy '' '' 
Fri Jan 29 2021 16:45:27 GMT+0800 (China Standard Time) 5.2 CloudBase Framework::info callHooks 'preDeploy' 
Fri Jan 29 2021 16:45:29 GMT+0800 (China Standard Time) 6.8 CloudBase Framework::info 🔧 init: deno...
Fri Jan 29 2021 16:45:29 GMT+0800 (China Standard Time) 6.8 CloudBase Framework::debug DenoPlugin: init {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.'
} 
Fri Jan 29 2021 16:45:29 GMT+0800 (China Standard Time) 6.8 CloudBase Framework::info 🔨 build: deno...
Fri Jan 29 2021 16:45:29 GMT+0800 (China Standard Time) 6.8 CloudBase Framework::debug DenoPlugin: build {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.'
} 
Fri Jan 29 2021 16:45:29 GMT+0800 (China Standard Time) 6.9 CloudBase Framework::debug ContainerPlugin: init {
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.',
  localAbsolutePath: '/Users/gpdi/cloudbase-framework/builds/cloudbase-deno-build-1611909929364/deno-app'
} 
Fri Jan 29 2021 16:45:30 GMT+0800 (China Standard Time) 8.0 CloudBase Framework::debug Container Plugin Inputs 'cloud' { cpu: 1, mem: 1, maxNum: 10, minNum: 1 } 'mode' {} 
Fri Jan 29 2021 16:45:30 GMT+0800 (China Standard Time) 8.0 CloudBase Framework::debug ContainerPlugin: build {
  uploadType: 'package',
  description: '基于云开发 CloudBase Framework 部署的云托管',
  isPublic: true,
  flowRatio: 100,
  cpu: 1,
  mem: 1,
  minNum: 1,
  maxNum: 10,
  policyType: 'cpu',
  policyThreshold: 60,
  containerPort: 80,
  dockerfilePath: './Dockerfile',
  buildDir: './',
  version: '1.0.0',
  localPath: './',
  envVariables: {},
  dockerImage: 'debian:latest',
  runtime: 'latest',
  denonVersion: '',
  entry: '',
  autoBuild: true,
  serviceName: 'deno-app',
  servicePath: '/deno-app',
  projectPath: '.',
  localAbsolutePath: '/Users/gpdi/cloudbase-framework/builds/cloudbase-deno-build-1611909929364/deno-app'
} 
Fri Jan 29 2021 16:45:31 GMT+0800 (China Standard Time) 8.7 CloudBase Framework::debug describeCloudBaseBuildService {
  DownloadHeaders: null,
  DownloadUrl: '',
  PackageName: 'deno-app.zip',
  PackageVersion: '1611909931',
  RequestId: '25c9166d-9280-436d-bbbe-d6cac3a0f2ab',
  UploadHeaders: null,
  UploadUrl: 'https://cloudaccess-run-1258016615.cos.ap-shanghai.myqcloud.com/home%2F922d0822225540ed9db967c5ef2bb198%2Fdeno-demo-3ghgaf7pa8d97b90-deno-app_1611909931_mgtM4vmG.zip?sign=q-sign-algorithm%3Dsha1%26q-ak%3DAKIDYqreAvj0McQ8bplDBtRUJcfwwlL299NL%26q-sign-time%3D1611909931%3B1612514731%26q-key-time%3D1611909931%3B1612514731%26q-header-list%3Dhost%26q-url-param-list%3D%26q-signature%3D60c6b9b535c70290fe089995fb16f887257f445f'
} 'deno-app' '/Users/gpdi/cloudbase-framework/builds/cloudbase-container-build-1611909929473/deno-app.zip' 
Fri Jan 29 2021 16:45:31 GMT+0800 (China Standard Time) 8.7 CloudBase Framework::error  Cannot read property 'reduce' of null
Fri Jan 29 2021 16:45:31 GMT+0800 (China Standard Time) 8.7 CloudBase Framework::info 部署日志: '/Users/gpdi/cloudbase-framework/logs/2021-01-29_16-45-22.log'

**Comment by binggg** [comment]
昨天后端服务有发布导致，回滚了
